src/Express.js

```diff
@@ -1,75 +1,82 @@
 'use strict';
 
+import {Router} from 'ferry';
```

Comment:
Yes, it took forever to figure out the import syntax. [Much better info here](http://babeljs.io/docs/usage/modules).